src/plugins/intel_cpu/src/transformations/transformation_pipeline.cpp

```diff
@@ -470,6 +471,34 @@ void Transformations::PreLpt(const std::vector<ov::element::Type>& defaultPrecis
         },
         ov::pass::NormalizeL2Decomposition);
 
+    CPU_SET_CALLBACK_X64(manager,
+        [this](const_node_ptr &node) -> bool {
+            if (node->is_dynamic() || node->get_element_type() != element::f32 || inferencePrecision != element::f32)
```

Comment:
Since `ov::pass::GroupNormalizationDecomposition` should be called before Snippets, LPT and `MVNDecomposition`, this callback is more about Snippets callback.
May I ask you please to add a few words about that?

Reply:
added, thanks Alexandra!